multiplatform-movies/android/src/main/java/com/novoda/movies/gallery/GalleryAdapter.kt

```diff
@@ -0,0 +1,45 @@
+package com.novoda.movies.gallery
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+import com.novoda.movies.R
+
+
+class GalleryAdapter(private val moviePosterWidthInPixels: Int) : RecyclerView.Adapter<GalleryItemViewHolder>() {
```

Comment:
I think nowadays we should be able to avoid having to have pixels measurements for images and grid to play well.
Usually tweaking the constraintlayout should work but I'm not 💯 % sure of the exact solution

Reply:
you could also override (or use [GridLayoutManager](https://developer.android.com/reference/android/support/v7/widget/GridLayoutManager))

Reply:
oh you do. Then I don't think you need to modify the layout params of the view holder manually?

Reply:
Unfortunately, we still have to do that as the default behavior of the `GridLayoutManager` is not to fill the space automatically :(